web-gateway/conf/routes

```diff
@@ -22,3 +22,5 @@ GET         /search                     controllers.SearchController.searchForm(
 POST        /search                     controllers.SearchController.search()
 
 GET         /assets/*file               controllers.Assets.at(path = "/public", file)
+
+GET         /*file                      controllers.ExternalAssets.at(path="../", file)
```

Comment:
hi @mehulsr we can't do this. With this route I can download any file in the folder including the source code (test http://localhost:9000/LICENSE or http://localhost:9000/build.sbt for example).

I think we can safely link to https://github.com/lagom/online-auction-java/blob/master/README.md  on the instructions (in which case this route won't be necessary).